### ISSUE TITLE
Maximum recursion depth exceeded

### ISSUE THREAD

**Issue by jonathanbalagot** [OpenPoGo/OpenPoGoBot#147]
Saw this issue here but it occurred twice for me using the latest release. - https://github.com/OpenPoGo/OpenPoGoBot/issues/135

Both errors seem to occur around 10-15 minutes after the bot runs. Will run it again.

Windows 10
Commit - e1bd866e7297663bb236efd719a4122543a4adc5

```
[2016-07-31 23:45:57] [#] Found fort The Pond  at distance 0.24km
[2016-07-31 23:45:57] [#] Moving closer to The Pond
[2016-07-31 23:45:57] [#] Walking from (40.76631063172019, -73.97579903011791) to (40.766147, -73.975779) for approx. 8.00 seconds
Traceback (most recent call last):
  File "C:\pgbotfork\pokecli.py", line 293, in <module>
    main()
  File "C:\pgbotfork\pokecli.py", line 286, in main
    bot.take_step()
  File "C:\pgbotfork\pokemongo_bot\__init__.py", line 76, in take_step
    self.stepper.take_step()
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 41, in take_step
    self._work_at_position(position[0], position[1])
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 137, in _work_at_position
    self.bot.work_on_cell(cell)
  File "C:\pgbotfork\pokemongo_bot\__init__.py", line 90, in work_on_cell
    walk_worker.work()
  File "C:\pgbotfork\pokemongo_bot\cell_workers\walk_towards_fort_worker.py", line 62, in work
    self.stepper.walk_to(self.config.walk, *position)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 68, in walk_to
    self._do_walk_to(speed, position_lat, position_lng, lat, lng, alt, 25)
 ...
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 94, in _do_walk_to
    self._work_at_position(position_lat, position_lng)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 137, in _work_at_position
    self.bot.work_on_cell(cell)
  File "C:\pgbotfork\pokemongo_bot\__init__.py", line 90, in work_on_cell
    walk_worker.work()
  File "C:\pgbotfork\pokemongo_bot\cell_workers\walk_towards_fort_worker.py", line 62, in work
    self.stepper.walk_to(self.config.walk, *position)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 68, in walk_to
    self._do_walk_to(speed, position_lat, position_lng, lat, lng, alt, 25)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 94, in _do_walk_to
    self._work_at_position(position_lat, position_lng)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 137, in _work_at_position
    self.bot.work_on_cell(cell)
  File "C:\pgbotfork\pokemongo_bot\__init__.py", line 90, in work_on_cell
    walk_worker.work()
  File "C:\pgbotfork\pokemongo_bot\cell_workers\walk_towards_fort_worker.py", line 62, in work
    self.stepper.walk_to(self.config.walk, *position)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 68, in walk_to
    self._do_walk_to(speed, position_lat, position_lng, lat, lng, alt, 25)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 94, in _do_walk_to
    self._work_at_position(position_lat, position_lng)
  File "C:\pgbotfork\pokemongo_bot\stepper.py", line 123, in _work_at_position
    response_dict = self.api_wrapper.call()
  File "C:\pgbotfork\api\__init__.py", line 96, in call
    results = request.call()
  File "c:\pgbotfork\src\pgoapi\pgoapi\pgoapi.py", line 177, in call
    request = RpcApi(self._auth_provider)
  File "c:\pgbotfork\src\pgoapi\pgoapi\rpc_api.py", line 56, in __init__
    self._session = requests.session()
  File "C:\Python27\lib\site-packages\requests\sessions.py", line 689, in session
    return Session()
  File "C:\Python27\lib\site-packages\requests\sessions.py", line 346, in __init__
    self.mount('https://', HTTPAdapter())
  File "C:\Python27\lib\site-packages\requests\adapters.py", line 109, in __init__
    self.init_poolmanager(pool_connections, pool_maxsize, block=pool_block)
  File "C:\Python27\lib\site-packages\requests\adapters.py", line 145, in init_poolmanager
    block=block, strict=True, **pool_kwargs)
  File "C:\Python27\lib\site-packages\requests\packages\urllib3\poolmanager.py", line 66, in __init__
    dispose_func=lambda p: p.close())
  File "C:\Python27\lib\site-packages\requests\packages\urllib3\_collections.py", line 47, in __init__
    self._container = self.ContainerCls()
  File "C:\Python27\lib\collections.py", line 69, in __init__
    self.__update(*args, **kwds)
  File "C:\Python27\lib\abc.py", line 151, in __subclasscheck__
    if subclass in cls._abc_cache:
  File "C:\Python27\lib\_weakrefset.py", line 72, in __contains__
    wr = ref(item)
RuntimeError: maximum recursion depth exceeded
```

**Comment by wchill** [comment]
#136 should fix this.

**Comment by adventurer** [comment]
can i add this in pokecli.php 

import sys  
sys.setrecursionlimit(1000000)

**Comment by wchill** [comment]
Yes, but it is not recommended.

**Comment by andythorne** [comment]
Fixed in #136
